src/YoutubeAtom.tsx

```diff
@@ -7,9 +7,9 @@ import { palette, space } from '@guardian/src-foundations';
 import { textSans } from '@guardian/src-foundations/typography';
 import { SvgPlay } from '@guardian/src-icons';
 
-import { YoutubeStateChangeEventType } from './types';
 import { MaintainAspectRatio } from './common/MaintainAspectRatio';
 import { formatTime } from './lib/formatTime';
+import { YoutubeStateChangeEventType } from './types';
```

Comment:
You could just inline this. I don't think it's being used anywhere else?

Reply:
it use to be used for unit tests, so yes good point ill move it back